tim/greet.js

```diff
@@ -0,0 +1,11 @@
+
+
+exports.greeting = function(name) {
+  console.log(arguments);
```

Comment:
You should get rid of console.logs that aren't part of functionality before submitting code.